setup.cfg

```diff
@@ -0,0 +1,56 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the 3-clause BSD style license (see LICENSE).
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019 Datadog, Inc.
+
+[metadata]
+name = apigentools
+version = 0.9.0.dev1
+description = Tooling for generation of API clients using openapi-generator
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords =
+    openapi
+    api
+    client
+    openapi-generator
+author = Slavek Kabrda
+author_email = slavek.kabrda@datadoghq.com
+url = https://github.com/DataDog/apigentools
+license = BSD 3-clause
+platforms = any
+classifiers=
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3.7
+
+
+[options]
+packages = find:
+include_package_data = True
+scripts =
+    container-apigentools
+install_requires =
+    chevron==0.13.1
+    PyYAML==5.1
+
+[options.entry_points]
+console_scripts =
+    apigentools = apigentools.cli:cli
+
+[options.extras_require]
+tests =
+    flexmock>=0.10.4
+    pytest>=5.2.4
+    pytest-cov
+
+[options.packages.find]
+exclude = tests
+
+[bdist_wheel]
+universal = 1
```

Comment:
Python 3 is required, no?

Reply:
You are right. It should be `py3` only wheel.